index.js

```diff
@@ -112,8 +114,13 @@ LiveServer.start = function(options) {
 	var wait = options.wait || 0;
 
 	// Setup a web server
-	var app = connect()
-		.use(staticServerHandler) // Custom static server
+	var app = connect(); 
+	options.mount.forEach(function(mountRule) {
+		var mountPath = path.resolve(process.cwd(), mountRule[1]);
```

Comment:
Maybe this resolve should be removed? I was considering the commandline args already to be absolute after the resolve in index.js, so this should only be relevant for the node API. But actually it is up to the developer to bring in some reasonable paths here.